app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/address/enter_postcode.scala.html

```diff
@@ -65,18 +65,26 @@
   }
 }
 
+@userMsgKey = @{
```

Comment:
something like `titleUserMessageKey` would be more descriptive here - it's hard to tell what the difference is between `userMsgKey` and `userKey`